modules/infra/submodules/cost-usage-report/main.tf

```diff
@@ -0,0 +1,55 @@
+data "aws_caller_identity" "aws_account" {}
+data "aws_partition" "current" {}
+
+locals {
+  private_subnet_ids = var.network_info.subnets.private[*].subnet_id
+
+  aws_account_id                = data.aws_caller_identity.aws_account.account_id
```

Comment:
Unless the locals defined are used in multiple places, I feel it makes it harder to read.

Reply:
Yes, in fact this is used in multiple places that's why we have it like this